cmd/kubeadm/app/preflight/checks_linux.go

```diff
@@ -42,3 +45,14 @@ func (idsc IsDockerSystemdCheck) Check() (warnings, errorList []error) {
 	}
 	return warnings, nil
 }
+
+// Check determines if IPVS proxier can be used or not
+func (ipvspc IPVSProxierCheck) Check() (warnings, errors []error) {
+	ipsetInterface := utilipset.New(ipvspc.exec)
+	kernelHandler := ipvs.NewLinuxKernelHandler()
+	_, err := ipvs.CanUseIPVSProxier(kernelHandler, ipsetInterface)
+	if err != nil {
```

Comment:
Since we are ignoring the other returned value this could be:
```go
if _, err := ipvs.CanUseIPVSProxier(kernelHandler, ipsetInterface); err != nil {
    return nil, append(errors, err)
}
```

Reply:
@detiber Thank you for the review! I've updated the PR.